docgenhtml/plugins/org.polarsys.capella.docgen.configuration.ui/src/org/polarsys/capella/docgen/configuration/ui/wizard/SelectElementsWizardPage.java

```diff
@@ -185,6 +189,17 @@ public void createControl(Composite parent) {
 		// update wizard status
 		dialogChanged();
 		setControl(container);
+		
+		if(initialSelection != null) {
+			leftTreeViewer.setSelection(initialSelection, true);
+			TreeSelection selectionInTree = (TreeSelection)leftTreeViewer.getSelection();
+			for(TreePath path : selectionInTree.getPaths()) {
+				leftTreeViewer.setChecked(path.getLastSegment(), true);
+			}
```

Comment:
i think it is better to move this to initialize method

Reply:
Moved it to initialize()